composer.json

```diff
@@ -27,7 +27,17 @@
   },
   "require": {
     "php": ">=7.0.0",
+    "ext-dom": "*",
+    "ext-libxml": "*",
+    "ext-simplexml": "*",
+    "leafo/scssphp": "^0.7.4",
```

Comment:
please remove we currently only use the bundled version - composer integration of compiling libs need a different handling